release-team/README.md

```diff
@@ -185,3 +185,19 @@ is undefined.
 ### During "Security" releases
 For all unplanned or embargoed releases
 - Facilitate security releases following the under the [Security Release Process](security-release-process-documentation/security-release-process.md)
+
+---
+
+## Milestone Maintainers
+
+Across release cycles, one of the best signals for [issue triage](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#milestones) is whether or not an issue or PR is actually targeted for the current milestone.
+
+To maintain up-to-date status on milestone inclusion, we rely on a set of Milestone Maintainers (members of the `milestone-maintainers` GitHub team) to apply the appropriate labels to issues / PRs. This is facilitated by `/milestone` commands and bot automation.
+
+Each release cycle, the current Release Team should seek to manage membership to the aforementioned GitHub team.
+
+The `milestone-maintainers` GitHub team should contain:
+- At least 2 SIG leads (either SIG Chairs or SIG Technical Leads) with Maintainer access to manage membership to the GitHub team (for their SIG)
```

Comment:
We'd be allow 31*2 people to add new members to this team?  That would certainly remove the release team as the bottleneck in updating team membership, but I thought gating team membership behind "ask the release team" was a forcing function to ensure the team wasn't growing recklessly, and contained members that had been vetted by the release team.